examples/contrib/mue/FactorMuE.py

```diff
@@ -84,7 +84,9 @@ def main(args):
         # Specific data split seed, for comparability across models and
         # parameter initializations.
         pyro.set_rng_seed(args.rng_data_seed)
-        indices = torch.randperm(sum(data_lengths)).tolist()
+        indices = torch.randperm(
+            sum(data_lengths), generator=torch.Generator(device=device)
+        ).tolist()
```

Comment:
Is this tested?  Any idea why the the old version isn't failing tests when running on PyTorch 1.9?